src/utility/ErrorMacros.hpp

```diff
@@ -0,0 +1,39 @@
+#include <stdexcept>
+
+#include "build/version.hpp"
+#include "utility/spdlog-fmt.hpp"
+
+// clang-format off
+// NOLINTBEGIN(cppcoreguidelines-macro-usage)
+
+// Only use this one for internal errors. Clearly invalid states that shouldn't happen.
+#define daiCheckIn(A) \
```

Comment:
Namespace "prefix" + ALL CAPS for defines
```suggestion
#define DEPTHAI_CHECK_IN(A) \
```

Reply:
ok will change to all caps